src/util/logger.js

```diff
@@ -110,12 +110,16 @@ export type CreateBunyanLogParams = {|
 export type CreateBunyanLogFn = (params: CreateBunyanLogParams) => Logger;
 
 export type CreateLoggerOptions = {|
-  createBunyanLog: CreateBunyanLogFn,
+  createBunyanLog?: CreateBunyanLogFn,
+  logStream?: ConsoleStream,
```

Comment:
@saintsebastian it looks that the changes applied on this file are not necessary anymore to make our assertions on the produced logs, am I right? in that case we can remove these changes from this PR.